src/openzaak/components/documenten/query/django.py

```diff
@@ -92,14 +93,15 @@ class InformatieobjectRelatedQuerySet(
 class ObjectInformatieObjectQuerySet(BlockChangeMixin, InformatieobjectRelatedQuerySet):
 
     RELATIONS = {
-        BesluitInformatieObject: ObjectTypes.besluit,
-        ZaakInformatieObject: ObjectTypes.zaak,
+        BesluitInformatieObject: ObjectInformatieObjectTypes.besluit,
+        ZaakInformatieObject: ObjectInformatieObjectTypes.zaak,
     }
 
     def create_from(self, relation: IORelation) -> [models.Model, None]:
         if isinstance(relation.informatieobject, ProxyMixin):
             return None
 
+        # FIXME?
```

Comment:
not sure what is happening here - @SilviaAmAm, can you take a look if this makes sense?

Reply:
I don't think changes need to be made here, as this will never be triggered for `VerzoekInformatieObjecten` because only external verzoeken are supported

Reply:
yeah, I saw the tests are passing, so that seems to confirm it. I see the PR comment is outdated now, but if not there yet, can you add that in a comment in the code?

Reply:
added in https://github.com/open-zaak/open-zaak/pull/1226/commits/894903cb35fe0cb91ca7fc0029c34244151bd283